### ISSUE TITLE
Random clicks on fairs

### ISSUE THREAD

**Issue by atereshkov** [Tune389/Tap-Titans-2-Progress-Bot#60]
I encountered such a problem that on the latest version bot sometimes clicks on the fairy.

I've tried to change the random touch coordinates, but it's still happend sometimes and I don't know exactly the moment when the click happens (maybe right after prestige).

// ----------- RANDOM ATTACK
:randomTouch
	rand #randomX 60 415
	rand #randomY **280** 420

I hoped that randomY click between 280 and 420 will help, but the problem still occur.

And also I've tried to use the code from #32 but it seems like not working now.

**Comment by atereshkov** [comment]
I found the problem.
1. The click on the fairy occurs when bot try to get offline gold

// ----------- LEVEL HEROES
:levelHeroes

	//collect gold
	**//touchDown 0 15 230
	//sleep #btnDelay
	//touchUp 0**

2. And when check boss fight:

elseif #enableClanShip == 1
		**//collect gold (click away heroe screen)
		touchDown 0 15 230
		sleep #btnDelay**
		touchUp 0
		sleep #btnDelay
		touchPress 0 80 105
		sleep #shipDelay
		#time = #time + #shipDelay
	endif

**Comment by GalatasXF** [comment]
the error is not to click on fairies, the error is: not to click to close in right time i think if you use this in :runActions can solve your problems
`sleep 100
	if #enablePremiumCollect == 0
		touchDown 0 200 630
	else
		touchDown 0 300 630
	endif
	sleep 100
	touchUp 0
	sleep 100
	
	#time = #time + 300`

**Comment by atereshkov** [comment]
@GalatasXF I don't think that I have such a problem. I was able to reproduce the bug with fairies clicks, but it should be already fixed by my code. Maybe your error is kind of another.
